.github/dependabot.yml

```diff
@@ -10,3 +11,23 @@ updates:
     reviewers:
       - "darrenvong"
       - "lucaslcode"
+
+  - package-ecosystem: "pip-compile"
```

Comment:
According to https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#package-ecosystem the value should be `"pip"`, not `"pip-compile"`.

I'm all for pinned versions, great!

Reply:
Thanks, not sure how I screwed that up!